coinFlip.py

```diff
@@ -0,0 +1,16 @@
+"""
+author: kalez43@github.com
+description: A coin flip generator utilizing the random module and a list.
+"""
+

```

Comment:
Should there be a space here

Reply:
pycharm said there should be a space lol